core/lib/spree.rb

```diff
@@ -0,0 +1,34 @@
+require 'active_support/core_ext/module/attribute_accessors'
```

Comment:
We can't name a file spree.rb, because of [the other file of the same name](https://github.com/solidusio/solidus/blob/master/lib/spree.rb)

Reply:
Hmmm, I like the idea of a `spree.rb` defining the `Spree` module. (independent of the rest of these changes). Can we look at killing the other one? 

I am not a big fan of the "solidus" gem itself doing much.

Reply:
That file doesn't do much, but we should definitely deprecate it before removing it, various extensions and likely stores, which have been upgraded from spree, are expecting that require to require all of Solidus.

Reply:
I am not super worried about deleting the original without deprecating as I don't think it would create any runtime errors in stores, and could be caught and fixed very easily by anyone reading the changelog.

But now that i sit down and think about it, I'm not 100% sure how/when solidus is usually required into a project.

Anecdotally I grep'd my code folder and we don't have any projects, either inherited or greenfield that do a `require 'spree'`.

Reply:
Yea, I just checked and it looks like we usually get solidus required as part of:
`Bundler.require(*Rails.groups)` in config/application.rb's.

I doubt anyone is going to get caught by this, even if they are an old spree upgrade.